weasyl/oauth2.py

```diff
@@ -26,8 +26,9 @@ def render_form(self, scopes, credentials, mobile, error=None,
         else:
             user = user_media = None
         credentials['scopes'] = scopes
+        detail_scopes = [scope for scope in SCOPES if scope['name'] in scopes]
```

Comment:
Hm. Since it's a sanity error, perhaps a scope not in SCOPES should be an error.